docker/depends/Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN apt-get update \
 # INSTALL DEPENDENCIES
 # ----------------------------------------------------------------------
 COPY pecan.depends /
-RUN bash /pecan.depends \
+RUN Rscript -e "install.packages(c('devtools','roxygen2'), repos = 'http://cran.rstudio.com')" \
```

Comment:
This will automatically pick up any future Roxygen releases. Is that desirable here, or should we pin it with `devtools::install_version("roxygen2", "7.0.2")` here like we do for Travis?